lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -58,30 +55,35 @@ class CategoryRenderer {
       auditEl.classList.add('lh-audit--manual');
     }
 
-    return this._populateScore(auditEl, audit.result.score, scoreDisplayMode, title, description);
+    this._populateScore(auditEl, audit.result.score, scoreDisplayMode);
+
+    if (audit.result.error) {
+      auditEl.classList.add(`lh-audit--error`);
+      const valueEl = this.dom.find('.lh-score__value', auditEl);
+      valueEl.textContent = 'Error!';
```

Comment:
not sure I love the `!` on the end, save that sucker for the category score ;)

Reply:
I thought felt a little less naked with the bang.

but i'm with the errors being bang-less.

![image](https://user-images.githubusercontent.com/39191/39457332-0fd31336-4ca2-11e8-8b73-8ed164551a04.png)
![image](https://user-images.githubusercontent.com/39191/39457335-13a0e51a-4ca2-11e8-8255-ba10d9d3221c.png)